tests/cli/test_rasa_test.py

```diff
@@ -148,42 +150,32 @@ def test_test_core_comparison(
 
 
 def test_test_core_comparison_after_train(
-    run_in_simple_project: Callable[..., RunResult]
+    run_in_simple_project: Callable[..., RunResult],
+    trained_rasa_model: str,
+    tmp_path: Path,
 ):
-    write_yaml(
-        {"language": "en", "policies": [{"name": "MemoizationPolicy"}]}, "config_1.yml"
-    )
 
-    write_yaml(
-        {"language": "en", "policies": [{"name": "MemoizationPolicy"}]}, "config_2.yml"
-    )
+    path = Path(tmp_path / "comparison_models")
+    path.mkdir()
+    assert path.exists()
```

Comment:
I'd remove this line as this only checks if the Python code `path.mkdir` works as expected
```suggestion
```

Reply:
all done now 😃